### ISSUE TITLE
[BUG] Address is displayed incorrectly in orders view in admin panel

### ISSUE THREAD

**Issue by kaysond** [evershopcommerce/evershop#347]
**Describe the bug**
The address in the orderEdit component is not displayed correctly

**To Reproduce**
1. Create an order
2. Click on the order in the admin panel

**Expected behavior**
Order should be displayed correctly

**Screenshots**
![image](https://github.com/evershopcommerce/evershop/assets/1147328/49abf847-e869-49fb-9209-7d841f07a419)

**Background (please complete the following information):**
 - NodeJS Version - 20.7.0
 - Postgres Version - 15.4
 - EverShop Version - rc7
 - OS: [e.g. Window, Ubuntu, Mac-OS] - Debian
 - Browser [e.g. Chrome, Safari] - FF

**Additional context**
https://github.com/evershopcommerce/evershop/blob/6e16f046e0b95efa16431a5ea41c22215273e9dd/packages/evershop/src/modules/oms/pages/admin/orderEdit/Customer.jsx#L37-L52

`shippingAddress.address1` does not show up at all.

This might be different based on locale, but for North America, you would want:
```
   <div> 
     <span>{shippingAddress.fullName}</span> 
   </div> 
   <div> 
     <span>{shippingAddress.address1}</span> 
   </div> 
   <div> 
     <span>{`${shippingAddress.city}, ${shippingAddress.province.name}`}</span> 
   </div> 
   <div> 
     <span>{`${shippingAddress.postcode}, ${shippingAddress.country.name}`}</span> 
   </div> 
   <div> 
     <span>{shippingAddress.telephone}</span> 
   </div> 
```

**Comment by vivekvking** [comment]
Hi @treoden can you please specify how exactly this is wrong here? The address is visible on the admin panel. I want to work on it but I don't understand what the exact issue is.

**Comment by treoden** [comment]
Hi @vivekvking ,

The issue is `address1` was missing from the Graphql Query at the below file:

https://github.com/evershopcommerce/evershop/blob/6e16f046e0b95efa16431a5ea41c22215273e9dd/packages/evershop/src/modules/oms/pages/admin/orderEdit/Customer.jsx#L124C7-L137C8

It happens to the `billingAddress` right below too. We need to add a new graphQL field `address1` of the `shippingAddress` and `billingAddress`

Thanks

**Comment by treoden** [comment]
Hi @vivekvking ,

Just want to confirm you are working on it and will submit a PR soon? I want to include this fix in the up coming release (this week)
Thanks

**Comment by treoden** [comment]
Hi @kaysond ,

This is fixed in `1.0.0-rc.8`. 

For the localization, you can modify the address summary by creating a component in your them following below path:

`<your theme>/components/common/customer/address/AddressSummary.jsx`

Thanks
